frontend/packages/data-portal/app/components/BrowseData/BrowseDataSearch.tsx

```diff
@@ -17,8 +17,15 @@ export function BrowseDataSearch() {
   const [query, setQuery] = useState(searchParams.get('search') ?? '')
 
   // If the user hasn't typed in a key for 500ms, then update the search params.
+  const initialLoadRef = useRef(true)
```

Comment:
Does this get initialized/re-initialized any time a user is navigating to search results (including clicking "Back to Results")?